src/main/resources/application.properties

```diff
@@ -40,4 +40,7 @@ requisitionUri=${REQUISITION_URI:/#!/requisition/{0}/fullSupply}
 time.zoneId=${TIME_ZONE_ID:UTC}
 email.noreply=${MAIL_ADDRESS:noreply@openlmis.org}
 
-authorization.skip=${SKIP_AUTHORIZATION:false}
\ No newline at end of file
+authorization.skip=${SKIP_AUTHORIZATION:false}
+
+cors.allowedOrigins=${CORS_ALLOWED_ORIGINS}
```

Comment:
Looks good, but remember to contribute these settings with a description and sample values to the config repo & the template service.